### DISCUSSION TITLE
How the pdf is generated?

### DISCUSSION THREAD

**Discussion by llaniewski** [krahets/hello-algo#1161] [Q&A]
Hi!

Great project. I really got inspired by it!

I would like to ask how exactly the PDF version is generated from the source?
Could anybody point me to the place in the repo or some tools that are used to generate the PDF? Is it some feature of `mkdocs`?

All the best,
Lukasz

**Comment by krahets** [comment]
Thanks for your interest!

We use Pandoc to convert markdown documents to PDF.
https://pandoc.org/#

**Reply by llaniewski** [reply]
Thanks! Are the scripts you're using public? I'm quite familiar with pandoc, knitr and R Markdown, but not much with `mkdocs`.
